src/components/Accordion/Accordion.tsx

```diff
@@ -77,24 +84,28 @@ export const AccordionSummary: FC<AccordionSummaryProps> = ({
   );
 
   return (
-    <div
-      aria-expanded={expanded}
-      aria-controls={`${id}-content`}
-      className={headerClassnames}
-      onClick={onClick}
-      onKeyDown={handleKeyDown}
-      id={`${id}-header`}
-      role="button"
-      tabIndex={0}
-      {...rest}
-    >
+    <div className={headerClassnames} id={`${id}-header`} {...rest}>
+      <div
```

Comment:
I have mixed feelings about this. Now it is less clear what the content is that is being expanded. And the reading from VoiceOver is quite confusing to me:
"Collapse content, expanded, button"
"Expand content, collapsed, button"
should we externalize the aria-label here?

In this case, what would we want/expect the button to announce?
![Screenshot 2024-04-24 at 2 58 45 PM](https://github.com/EightfoldAI/octuple/assets/106610186/d3d0dcf7-db77-43e5-9909-0cfe01b94a3d)

Reply:
The aria labels are externalized via props, but we need to ensure there is a default label regardless.

Reply:
As the Header content is a `React. ReactNode`, there's no guarantee it's a `string`. I can make an update applying the header content to the aria label if it's a `typeof` `string`, then fallback to the default aria labels. That should suffice for basic implementations. Ideally the aria labels will be provided at implementation time via props, but engineers may miss this one.

Reply:
For example, in the screen above, the content isn't a string. It contains 2 elements that in turn, contain strings, and from an aria-label perspective, would not be valid.

Reply:
Pushed updates:

Content design feedback:
Open content, collapsed, button
Close content, expanded, button

Also reversed the default/externalized aria labels such that they are more intuitively aligned with the state of the buttons:

`collapseAriaLabelText` = "Close content"
`expandAriaLabelText` = "Open content"

Reply:
The hard truth is, in order to be 100% accessible while retaining the same level of useful information, aria labels will need to be provided as an implementation task.

Reply:
Another option would be to only pass the component name as the default aria label:

Content design feedback:
Accordion, collapsed, button
Accordion, expanded, button

LMK